packages/bpk-component-nudger/src/BpkConfigurableNudger.tsx

```diff
@@ -82,32 +85,47 @@ const BpkConfigurableNudger = ({
     inputClassName && inputClassName,
     buttonType === 'secondaryOnDark' && 'bpk-nudger__input--secondary-on-dark',
   );
+  const inputRef = useRef<HTMLInputElement | null>(null);
 
   return (
     <div className={classNames}>
       <BpkButtonV2
         type={BUTTON_TYPES[buttonType]}
         iconOnly
-        onClick={() => onChange(decrementValue(value))}
+        onClick={() => {
+          const newValue = decrementValue(value);
+          onChange(newValue);
+          // We want to maintain native input events across our form components. Along with react updating
+          // the value attribute we can set it via native handlers and emit a "change" event.
+          inputRef.current && setNativeValue(inputRef.current, `${newValue}`);
+        }}
         disabled={minButtonDisabled}
         title={decreaseButtonLabel}
         aria-controls={id}
-      >        
+      >
         <AlignedMinusIcon />
       </BpkButtonV2>
       <input
         type="text"
         aria-live="polite"
-        readOnly
-        value={formatValue(value)}
+        defaultValue={formatValue(value)}
         id={id}
+        ref={inputRef}
+        name={name || id}
+        onChange={(event) => onChange(parseInt(event?.target.value, 10))}
```

Comment:
This ensures that we're able to parse the user input as a number. Otherwise if user enters "1" and we increment the value we get "11" 🎖️ Javascript at it's best.

Reply:
Perhaps it has to happen further up?